### PR TITLE
build: remove enzyme after rtl migration

### PR DESCRIPTION
**Pull request by PavelKirvelTextkernel**
Task [link](https://textkernel.atlassian.net/browse/ONEUI-414)

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -24,8 +24,7 @@
         "prepublishOnly": "npm run check-npm-publish-access; npm run tsc; npm run test; npm run lint:ci; npm run lint:styles; npm run build:prod",
         "coveralls": "cat ./coverage/lcov.info | node node_modules/.bin/coveralls",
         "release": "standard-version -t ''",
-        "prepare": "husky install",
-        "measure-enzyme-to-rtl": "node --experimental-fetch scripts/measure-enzyme-to-rtl-tests-ratio/measure-enzyme-to-rtl-tests-ratio.js"
```
Comment (antipin) [inline package.json:28]: Let's also remove the script itself `scripts/measure-enzyme-to-rtl-tests-ratio/measure-enzyme-to-rtl-tests-ratio.js`

File: scripts/test/enzyme.setup.js [hunk 2]
```diff
@@ -1,12 +0,0 @@
-/* eslint import/no-extraneous-dependencies: [error, { devDependencies: true }] */
-
-import 'regenerator-runtime/runtime';
```
Comment (antipin) [inline scripts/test/enzyme.setup.js:3]: Seems like `regenerator-runtime` package is not used anywhere else, so we can remove it as well

File: package.json
Comment (antipin) [inline package.json:1]: There are still a few places that are overlooked:
* a rule for `enzyme` in `renovate.json` that must be removed
* types in `src/types/enzyme.d.ts`
* `CONTRIBUTING.md` mentions enzyme once
* `TESTING_LIBRARY_MIGRATION.md` can be removed I think (cc @eszthoff)

**Review by antipin — CHANGES_REQUESTED** [review]

File: package.json (hunk 1 above)
Comment (PavelKirvelTextkernel) [reply package.json:28]: :+1:

File: scripts/test/enzyme.setup.js (hunk 2 above)
Comment (PavelKirvelTextkernel) [reply scripts/test/enzyme.setup.js:3]: :+1:

File: package.json
Comment (PavelKirvelTextkernel) [reply package.json:1]: :+1:

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -10,7 +10,6 @@ Any new implementation is expected to meet the following standards before it can
 -   The component has a [**displayName**](#display-names) defined.
 -   The implementation complies with [**accessibility**](#accessibility) standards.
 -   The component is **properly exported** in `src/index.js` (maintain alphabetical order).
--   The implementation is thoroughly [**covered by tests**](#testing) (Jest / Enzyme, preferably 100%), not only with snapshots.
```
Comment (antipin) [inline CONTRIBUTING.md:13]: This line still makes sense, just replace `Jest / Enzyme` with `Jest`

**Review by antipin — CHANGES_REQUESTED** [review]

File: CONTRIBUTING.md (hunk 3 above)
Comment (PavelKirvelTextkernel) [reply CONTRIBUTING.md:13]: :+1:

**Pull request merged**